components/metype-commenting-widget.js

```diff
@@ -6,11 +6,12 @@ class MetypeCommentingWidget extends React.Component {
   constructor(props) {
     super(props);
     this.randomNumber = new Date().getMilliseconds();
+    this.metypeWidget = React.createRef();
   }
 
   componentDidMount() {
-    !window.talktype && scriptLoader(this.props.host, () => this.initWidget(this.randomNumber));
-    this.initWidget(this.randomNumber);
+    !window.talktype && scriptLoader(this.props.host, () => this.initWidget());
+    this.initWidget();
```

Comment:
why is the random number removed, this might fail in infinite scroll

Reply:
randomNumber is still existed in <div> while rendering it, here I removed it because it was used while `getElementById` where I have replaced it by `ref` instead.